.vscode/settings.json

```diff
@@ -23,5 +23,6 @@
   "typescript.tsdk": "node_modules\\typescript\\lib",
   "[json]": {
     "editor.defaultFormatter": "esbenp.prettier-vscode"
-  }
+  },
```

Comment:
Can we enable this at a smaller scope, say only for the `eng` folder but not the rest of the repo?